src/test/resources/insurance_portal_demo_test.xml

```diff
@@ -608,6 +608,7 @@
         <values>3</values>
         <values>4</values>
         <values>viac ako 4</values>
+        <init>1</init>
```

Comment:
why was this added?

if this change was needed to pass some existing unit test, then it might hint at a bug caused by the change to the `EventService`